Automation/General/Way of working.md

```diff
@@ -248,7 +262,22 @@ Bad branch names:
 
 `itisdifficulttoreadthenameofthisbranch`
 
-#### Pull requests
+#### Pull requests (PR)
+
+**Opening a PR:**
+
+- Before diving into doing huge changes in the repository, discussed them with your team
+- Keep PRs small and focused
+- Add a description why the PR is necessary
+- Split commits into logical sections
```

Comment:
E.g. if tests and configuration files are being changed, better to have separate branches for that.
If it has to be on the same branch because configuration and test changes are connected in some way, then use different commit messages instead just one that covers 10 different files with the same message when obviously there are some specifics in each change.

Also makes sense to apply that to feature.
Make smaller PRs that change only tests related to a single feature or a few very related ones.
Just to avoid having a 15, 20 or more changed files, especially if there are a lot of changes.
Even one changed file is too much to review if it has 500 lines of code.

It is tricky to describe this more specifically in short bullets.
The point is not to be super specific and say something is wrong or more wrong.
In the end it is up to the developer. The point is just to keep it small(ish), simple and add a description when needed.

Reply:
At first, I found it ambiguous and didn't know if you meant the (group) order of commits or the changes that are "stored" within single commits. Now I understand that you meant the latter. Thanks for clarifying. :)